apps/clappr.io/docs/faq.md

```
@@ -63,11 +63,11 @@ Clappr.Log.setLevel(Clappr.Log.LEVEL_DEBUG) // or Clappr.Log.setLevel(0)
 
 Format         |HLS|MP4|MP3|WEBM| DASH | RTMP | FLV |
 ---------------|---|---|---|----|------|------|-----|
-Microsoft Edge | ✅ | ✅ | ✅ | ✅ | [🔌](/packages/dash-shaka-playback) | [🔌](https://github.com/clappr/clappr-rtmp-plugin) | [🔌](https://github.com/andrefilimono/clappr-flvjs-playback)
-Mozilla Firefox| ✅ | ✅ | ✅ | ✅ | [🔌](/packages/dash-shaka-playback) | [🔌](https://github.com/clappr/clappr-rtmp-plugin) | [🔌](https://github.com/andrefilimono/clappr-flvjs-playback)
-Google Chrome  | ✅ | ✅ | ✅ | ✅ | [🔌](/packages/dash-shaka-playback) | [🔌](https://github.com/clappr/clappr-rtmp-plugin) | [🔌](https://github.com/andrefilimono/clappr-flvjs-playback)
-Android        | ✅ | ✅ | ✅ | ❌ | [🔌](/packages/dash-shaka-playback) | [🔌](https://github.com/clappr/clappr-rtmp-plugin) | [🔌](https://github.com/andrefilimono/clappr-flvjs-playback)
-Safari         | ✅ | ✅ | ✅ | ❌ | [🔌](/packages/dash-shaka-playback) | [🔌](https://github.com/clappr/clappr-rtmp-plugin) | [🔌](https://github.com/andrefilimono/clappr-flvjs-playback)
+Microsoft Edge | ✅ | ✅ | ✅ | ✅ | [🔌](https://github.com/clappr/clappr/tree/main/packages/dash-shaka-playback) | [🔌](https://github.com/clappr/clappr-rtmp-plugin) | [🔌](https://github.com/andrefilimono/clappr-flvjs-playback)
```

markdownlint-cli2 (0.23.2)

[warning] Table pipe style
Expected: leading_and_trailing; Actual: no_leading_or_trailing; Missing leading pipe

(MD055, table-pipe-style)